lib/get-file-paths.js

```diff
@@ -4,7 +4,7 @@ var path = require('path');
 
 /**
  * Returns an object of file path data
- * 
+ *
  * @param {object}  file  a vinyl file object
  */
 exports.getFilePaths = function (file) {
```

Comment:
@ericduran open to any other way to do this. 

goal is to have:

```
var paths = {
     absolute: // absolute, full path to file
     relative: // relative path to file,
     folder: // relative path to file's containing directory,
     directory: // the actual directory name
   }
```

all four variables were needed in multiple places, multiple times, so I write this to get all of them at once and have them available throughout the system - main goal is to be as DRY as possible